### DISCUSSION TITLE
Browser access from another box

### DISCUSSION THREAD

**Discussion by Jacek-ghub** [madMAx43v3r/mmx-node#81] [General]
Would it be possible to bind port 11380 to the box IP address instead of 127.0.0.1?  I assume that some people will have distributed MMX installations (node + few harvesters + wallet) and using RDP or ssh port tunneling is rather a clunky way (comparing to a browser with a few tabs).  Although, that would mean going with SSL to protect such connection from snooping.

This way, we could basically lock those boxes completely with eventually opening only the few needed ports.

**Comment by rhernaus** [comment]
You can edit `config/local/HttpServer.json` and change `"host": "localhost"` to `"host": "0.0.0.0"`

**Comment by Jacek-ghub** [comment]
Super!  Thank you, works like a charm.

Now that you mentioned it, I have also found it in mmx-faq on discord!

By the way, do you know whether the data exchanged is encrypted, or just password is used to gain an access (and Wire Shark can be used to tap on that connection)?

Also, it would be really nice, if a different password access levels would be there.  For instance "read only" that would block access to wallet transactions, ...
